lessons/networking/networking-networking-hardware.md

```diff
@@ -1 +1,89 @@
-# 
\ No newline at end of file
+# [Networking] Hardware
+*Understanding components of a common home router*
+
+![Wi-Fi Router](.rsrc/wifi-router.jpeg)
+
+# Objectives
+- Understand common components of a home router
+- Learn network specific terminology
+- Learn what components could be useful to solve connection requirements
+
+# Introduction
+A total of 5 billion people around the world use the internet today based on report from April 2022. Many people are familar with a device called a router that allows them to connect to the internet. Home routers exist in various different forms and in this lesson you will learn what they represent. 
```

Comment:
Make this section more interesting/fun for the reader. Illustrate why they should care about identifying and understanding components of a router.

i.e. Networking hardware allow different devices to talk to one another at near instantaneous speeds. These technologies form the physical backbone of the internet

Reply:
Added more detail